doc/examples/anim13/anim13.sh

```diff
@@ -0,0 +1,69 @@
+# https://youtu.be/xxxxxxxxxx
+# The 66-sec, 1081 frame movie took 12 minutes to render on a 24-core MacPro 2013.
+
+# 0. Create an include file with key parameters
+cat << EOF > inc.sh
+TIME_RANGE=2020-10-06T05:55:10/2020-10-06T05:56:15
+PLOT_DOMAIN=2020-10-06T05:55:10/2020-10-06T05:56:15/-2.6/2.8
+DPI=200i
+SPLINE=linear
+EOF
+# 1. Create a dense point-file from the waveform data
+cat << EOF > pre.sh
+gmt begin
+	# 1a. Create dense point-files from the waveform data components
+	gmt psevents waveform_AV.DOL.txt -R\${PLOT_DOMAIN} -JX20cT/3.5c -Ar\${DPI} -i0,1+s1e-6,0 -f2T --GMT_INTERPOLANT=\${SPLINE} > E.txt
+	gmt psevents waveform_AV.DOL.txt -R\${PLOT_DOMAIN} -JX20cT/3.5c -Ar\${DPI} -i0,2+s1e-6,0 -f2T --GMT_INTERPOLANT=\${SPLINE} > N.txt
+	gmt psevents waveform_AV.DOL.txt -R\${PLOT_DOMAIN} -JX20cT/3.5c -Ar\${DPI} -i0,3+s1e-6,0 -f2T --GMT_INTERPOLANT=\${SPLINE} > Z.txt
+	# 1b. Setup the desired output times (24 frames per second)
+	gmt math -T\${TIME_RANGE}/24+i -o0 T --TIME_UNIT=s --FORMAT_CLOCK_OUT=hh:mm:ss.xxxxx = times.txt
+	# 1c. Create background page with location map and metadata
+	gmt text -R0/4.5/0/13 -Jx1c -F+j+f -B0 -X0.25c -Y0.25c <<- EOF
+	2.25 12.5 CM 18p 2020-10-06
+	2.25 11.7 CM 9p SE of Sand Point, Alaska
+	2.25 11 CM 10p 54.844@.N, 159.842@.W
+	2.25 10.5 CM 10p 31.0 km, Mw 6.0
+	2.25 10.0 CM 10p 2020-10-06 05:54:50 UTC
+	EOF
+	gmt grdimage -R190W/130W/30/75 @earth_relief_06m -Ba30+f -JM3.5c -I+d -Cterra --MAP_FRAME_TYPE=plain -X0.75c -Y0.5c -BWNbr --MAP_ANNOT_OBLIQUE=lat_parallel --FONT_ANNOT_PRIMARY=9p
+	echo 159.842W 54.844N | gmt plot -Sa0.5c -Gred -Wfaint
+	gmt meca -R190W/130W/30/75 -JM3.5c -Sm2.5c -M -G110/168/255 -T0 -W0.5p -Yh+0.5c <<- EOF
+	-159.70 54.81 39 0.82 -0.68 -0.14 0.63 0.34 -0.40 25
+	EOF
+gmt end show
+EOF
+# 2. Set up main movie script
+cat << EOF > main.sh
+gmt begin
+	gmt set FORMAT_DATE_MAP "o dd yyyy" FORMAT_CLOCK_MAP hh:mm FONT_ANNOT_PRIMARY +9p TIME_INTERVAL_FRACTION 0.01 FORMAT_CLOCK_MAP hh:mm:ss
+		gmt subplot begin 3x1 -Fs18.25c/3.5c -A -M0 -R\${PLOT_DOMAIN} -X5.5c -Y0.75c
+			gmt subplot set 0 -A"Z"
+			gmt plot waveform_AV.DOL.txt -Bpxafg -Bsxa1D -Bpyaf -BWSrt -W0.5p,darkgray -i0,3+s1e-6
+			gmt events Z.txt -Sc0.5p -Gred -Es+d0.5+r0.5 -Ms4+c1 -Mi0.5 -Mt+c0 -T\${MOVIE_COL0}
+			printf "%s -2.6\n%s 2.8\n" \${MOVIE_COL0} \${MOVIE_COL0} | gmt plot -W0.5p
+			gmt subplot set 1 -A"N"
+			gmt plot waveform_AV.DOL.txt -Bpxafg -Bsxa1D -Bpyaf -BWSrt -W0.5p,darkgray -i0,2+s1e-6
+			gmt events N.txt -Sc0.5p -Ggreen -Es+d0.5+r0.5 -Ms4+c1 -Mi0.5 -Mt+c0 -T\${MOVIE_COL0}
+			printf "%s -2.6\n%s 2.8\n" \${MOVIE_COL0} \${MOVIE_COL0} | gmt plot -W0.5p
+			gmt subplot set 2 -A"E"
+			gmt plot waveform_AV.DOL.txt -Bpxafg -Bsxa1D -Bpyaf -BWSrt -W0.5p,darkgray -i0,1+s1e-6
+			gmt events E.txt -Sc0.5p -Gblue -Es+d0.5+r0.5 -Ms4+c1 -Mi0.5 -Mt+c0 -T\${MOVIE_COL0}
+			printf "%s -2.6\n%s 2.8\n" \${MOVIE_COL0} \${MOVIE_COL0} | gmt plot -W0.5p
+		gmt subplot end
```

Comment:
No need to indent the whole "gmt subplot" block.

Reply:
No need to do any indention (not Python) but seems OK to keep that separate.
BTW, I removed the ss.xxxxx I had on the movie call and now I get whole seconds so I need to add the same fix I did in events to movie.  Maybe tomorrow before this is all resolved.

Reply:
> No need to do any indention (not Python) but seems OK to keep that separate.

Yes, my above comment is a little misleading. What I mean is the indentation of Line 52 (`gmt set`) and the "gmt subplot" block. To me, it's more readable that "gmt set" and "gmt subplot begin" are in the same indentation level.

This is the current indentation style:
```
gmt begin 
    gmt set 
        gmt subplot begin
            gmt subplot set 0
            ...
        gmt subplot end
gmt end 
```

What I prefer to is:
```
gmt begin 
    gmt set 
    gmt subplot begin
        gmt subplot set 0
        ...
    gmt subplot end
gmt end 
```

Reply:
Yes, my bad.  I will fix since doing the other thing as well.